sources/packages/web/src/services/AuthService.ts

```diff
@@ -24,6 +24,7 @@ export default async function(
     });
     if (keycloak.authenticated) {
       store.dispatch("auth/login", keycloak);
+      store.dispatch("student/setStudentProfileData",keycloak);
```

Comment:
Please add a space before "keycloak".

Reply:
running npm run lint on the whole web code base resolved this, but i dont know why this PR view on git hub isnt showing the space

Reply:
running npm run lint on the whole web code base resolved this, but i dont know why this PR view on git hub isnt showing the space